### DISCUSSION TITLE
Cover cropped nearly half

### DISCUSSION THREAD

**Discussion by 1301091** [PaulWoitaschek/Voice#1885] [Ideas]
Hi,

Thanks for the great app. I have one problem, however. I have high-quality square covers (usually 1200x1200) they were cropped a bit on my Pixel 3a, but on my new Galaxy A71, they are cropped to nearly half size. I attach two screenshots to show it. (I don't use the built-in cover downloader, because that too crops the images a bit. I only show the second image to see how bigger is the original cover.)

Thanks. I wish you the best.

![Screenshot_20200514-125827_Voice](https://user-images.githubusercontent.com/57151962/81931751-e11ecf00-95ea-11ea-88fc-44419386f17a.jpg)
![Screenshot_20200514-130029_Voice](https://user-images.githubusercontent.com/57151962/81931753-e1b76580-95ea-11ea-957e-b35eada4ed0d.jpg)

**Comment by Artyom-K** [comment]
Any update on this? For most of my books the actual title of the book at the top of the cover gets cropped
![Screenshot_20230316-103737](https://user-images.githubusercontent.com/6235266/225524879-81454a3f-2a51-4148-b346-1e851eec9b18.png)
I use the latest version of the app from here.
